### PR TITLE
#254: Added asterik(*) for the required fields

### PR DESCRIPTION
**Pull request by vbadkar**
Steps to test:
* Go to all the forms on the site.
* Check if the required fields input box has * Field required text below it as shown in below image.
image

### CODE REVIEW

File: assets/scss/main.scss [hunk 1]
```diff
@@ -24,4 +24,9 @@ body {
 .region-wrapper {
     display: flex;
     padding: 10px;
+}
+.required::after{
+    content: "*Field required";
+    color: red;
+    font-size: 12px;
 }
```
Comment (Libbna) [inline assets/scss/main.scss]: no new line at EOF. Please add a new line.

File: src/Controllers/CustomBlock.php [hunk 2]
```diff
@@ -1,64 +1,78 @@
...
+    $variables = parent::preprocesspage();
+    $displayBlockList = new Database();
+    $result = $displayBlockList->displayBlock();
+    $variables['result'] = $result;
+    if (isset($_SESSION["user_id"])) {
+      $variables['username'] = $_SESSION['username'];
+      $variables['authenticated_userId'] = $_SESSION['user_id'];
+      $variables['role'] = $_SESSION['role'];
     }
-}
```
Comment (Libbna) [inline src/Controllers/CustomBlock.php:64]: `no new line at EOF`

**Review by Libbna — CHANGES_REQUESTED** [review]

**Review by omkar-pd — APPROVED** [review]
Working fine!
image

**Review by Libbna — APPROVED** [review]
image

LGTM.

**Pull request merged**